cmd/snap/cmd_buy.go

```diff
@@ -100,7 +100,7 @@ func buySnap(opts *store.BuyOptions) error {
 	}
 
 	// TODO Remove this payment method filter once interactive payment methods are supported on the CLI
-	methods := make([]*store.PaymentMethod, 0)
+	var methods []*store.PaymentMethod
```

Comment:
it's grown via `append`, and exits if its length is zero. No, it doesn't matter.

And: this change was suggested by golint.